test/e2e/testing-manifests/statefulset/mysql-upgrade/statefulset.yaml

```diff
@@ -140,7 +140,7 @@ spec:
   - metadata:
       name: data
       annotations:
-        volume.alpha.kubernetes.io/storage-class: default
+        volume.beta.kubernetes.io/storage-class: default
```

Comment:
> But I wonder if we still want to keep the beta annotations in 1.8? 

The field has existed for 2 versions. I think we should remove the beta annotations, as they are misleading and nonfunctional as of this PR